main/src/main/java/cgeo/geocaching/activity/TabbedViewPagerActivity.java

```diff
@@ -37,6 +38,7 @@ public abstract class TabbedViewPagerActivity extends AbstractActionBarActivity
     private long[] orderedPages;
     private ViewPager2 viewPager = null;
     private Action1<Long> onPageChangeListener = null;
+    public OfflineTranslateUtils.Status translationStatus = new OfflineTranslateUtils.Status();
```

Comment:
I assume this could be "protected". I would actually put it directly in using Activity, but it is ok to have it here.

Still unsolved: values of this object need to be preserved on Activity recreation (using save bundle) eg on screen orientation change.

Reply:
no, needs to be public for the lambdas